packages/design-system/src/styles/_base.scss

```diff
@@ -139,12 +139,24 @@ $breakpoints: (
   font-family: var(--typography-heading__font-family);
   font-weight: var(--typography-heading__font-weight);
   line-height: var(--font-line-height-heading);
+  margin: 0;
+}
+
+@mixin heading-margins {
+  margin-block: 1.5em 0.5em;
+
+  &:first-child {
```

Comment:
Suggestion: Remove the styles within `:first-child` and `:last-child`.

I've had the most questions around this decision and it does seem to introduce issues for teams when using singular headings on a page. This was written under the assumption our headings would be used with other blocks of text.

Reply:
Thanks for the reminder. I think it makes sense to remove it. I'll do it in a follow-up PR

Reply:
It actually looks like [this line was the same in v6](https://github.com/CMSgov/design-system/blob/release/6.0/packages/design-system/src/styles/_base.scss#L162-L168). Did you mean [this line in the reset that applies to plain headings](https://github.com/CMSgov/design-system/blame/pwolfert/revert-heading-spacing-changes/packages/design-system/src/styles/_reset.scss#L65)?

Reply:
Oh! Yeah, I think I did get those mixed up!

Reply:
Updated in b4dd0b90c733f69d6e9df04e3b58a2325dbca8f1